packages/react-dom/src/events/ReactDOMEventListener.js

```diff
@@ -428,6 +438,13 @@ export function getEventPriority(domEventName: DOMEventName): * {
     case 'mouseenter':
     case 'mouseleave':
       return InputContinuousLanePriority;
+    case 'message': {
+      // We might be in the Scheduler callback.
+      // Eventually this mechanism will be replaced by a check
+      // of the current priority on the native scheduler.
+      const schedulerPriority = getCurrentPriorityLevel();
+      return schedulerPriorityToLanePriority(schedulerPriority);
```

Comment:
Can you add a TODO to inline schedulerPriorityToLanePriority into this file once you delete the flag? So that we can delete it from `SchedulerWithReactIntegration`. The only other place it's used is in the `else` branch of your feature flag.